modules/tinymce/src/core/main/ts/api/EditorUpload.ts

```diff
@@ -13,10 +13,26 @@ import * as Options from './Options';
 import { createUploader, openNotification } from './util/ImageUploader';
 
 /**
+ * TinyMCE Editor Upload API
  * Handles image uploads, updates undo stack and patches over various internal functions.
  *
- * @private
  * @class tinymce.EditorUpload
+ * @example
+ * // Apply a filter to all images as uploaded..
+ * tinymce.activeEditor.EditorUpload.addFilter((image) => {
+ *   const maxSize = 1920 * 1080;
+ *   const imageSize = image.width * image.height;
+ *   return imageSize < maxSize;
+ * });
+ *
+ * // UploadImages
+ * tinymce.activeEditor.EditorUpload.uploadImages();
+ *
+ * // ScanForImages
+ * tinymce.activeEditor.EditorUpload.scanForImages();
```

Comment:
```suggestion
 * // Upload images
 * tinymce.activeEditor.EditorUpload.uploadImages();
 *
 * // Scan for images
 * tinymce.activeEditor.EditorUpload.scanForImages();
```

If the associated comment is going to be just the API name, may as well edit said name for readability and legibility.

Reply:
These should be more descriptive, I'll fix them